<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="NIKHIL-PANDE" failures="1" tests="11" name="Test" time="0.252" errors="0" timestamp="2022-12-12T23:15:34 IST">
  <testcase classname="yt_TestNG.SoftAssert123" name="validation" time="0.022">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at yt_TestNG.SoftAssert123.validation(SoftAssert123.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- validation -->
  <testcase classname="yt_TestNG.I_TestContext" name="setUp" time="0.011"/>
  <testcase classname="yt_TestNG.I_TestContext" name="Logic" time="0.003"/>
  <testcase classname="yt_TestNG.Basic" name="launchUrl" time="0.01"/>
  <testcase classname="yt_TestNG.Basic" name="loginPage" time="0.008"/>
  <testcase classname="yt_TestNG.Basic" name="homePage" time="0.003"/>
  <testcase classname="yt_TestNG.Basic" name="logoutPage" time="0.007"/>
  <testcase classname="yt_TestNG._1DataProvider" name="dataMatrix" time="0.022"/>
  <testcase classname="yt_TestNG._1DataProvider" name="dataMatrix" time="0.011"/>
  <testcase classname="yt_TestNG._1DataProvider" name="dataMatrix" time="0.004"/>
  <testcase classname="yt_TestNG._1DataProvider" name="T2" time="0.001"/>
</testsuite> <!-- Test -->
